### PR TITLE
Issue templates: Upgrade to GitHub Forms

### PR DESCRIPTION
**Pull request by medikoo**
Proposed configuration is reflected here: https://github.com/medikoo/package-local-test/issues/new/choose

There's no support for forms in PR template, so updated just issue templates

This should limit amount of incomplete reports we deal with

@mnapoli @pgrzesik let me know what you think

### CODE REVIEW

**Review by pgrzesik — APPROVED** [review]
This looks great :tada: So much better than what we currently have

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  - type: checkboxes
+    attributes:
+      label: Are you certain there's a bug in the Framework?
+      description: If you're uncertain, please report at https://github.com/serverless/serverless/discussions instead
+      options:
+        - label: Yes it looks as a definite bug
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Is the issue in the Framework itself, or maybe it's influenced by external plugin?
```
Comment (mnapoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml:13]:
```suggestion
      label: Is the issue caused by a plugin?
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+      description: If it comes from the plugin, please report at given plugin issue tracker
```
Comment (mnapoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml:14]:
```suggestion
      description: If it comes from an external plugin, please report in the plugin issue tracker
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      options:
+        - label: It looks as an issue in the Framework itself
```
Comment (mnapoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml:16]:
```suggestion
        - label: It is not a plugin issue
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Are you using the latest version of the Framework
```
Comment (mnapoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml:20]:
```suggestion
      label: Are you using the latest version?
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,75 @@
+name: 🐛 Bug report
+description: Create a bug report
+body:
+  - type: checkboxes
+    attributes:
+      label: Are you certain there's a bug in the Framework?
```
Comment (mnapoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml:6]:
```suggestion
      label: Are you certain it's a bug?
```

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+      description: Latest version can be checked at https://github.com/serverless/serverless/releases/latest
+      options:
+        - label: Yes, I'm using the latest version
+          required: true
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists
+      options:
+        - label: I have searched the existing issues, and found it's not reported yet
```
Comment (mnapoli) [inline .github/ISSUE_TEMPLATE/bug-report.yml:30]:
```suggestion
        - label: I have searched existing issues, it hasn't been reported yet
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 7]
```diff
@@ -0,0 +1,21 @@
+name: 🎉 Feature request
+description: Suggest an idea
+body:
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists
+      options:
+        - label: I have searched the existing issues, and found it's not reported yet
```
Comment (mnapoli) [inline .github/ISSUE_TEMPLATE/feature-request.yml:9]:
```suggestion
        - label: I have searched existing issues, it hasn't been reported yet
```

**Review by mnapoli — COMMENTED** [review]
That should be super useful, even as a user (the existing template is useful but sometimes hard to use correctly).

I'm a bit afraid that the form may be a bit long, but we'll see if it's a problem.

I've added some inline suggestions.

**Comment by medikoo** [comment]
Thanks @mnapoli ! I've applied all suggestions

**Review by mnapoli — APPROVED** [review]
Awesome, LGTM!

**Pull request merged**